src/io/gff.rs

```diff
@@ -15,56 +15,94 @@
 //! ```
 //! use std::io;
 //! use bio::io::gff;
-//! let reader = gff::Reader::new(io::stdin());
+//! let reader = gff::Reader::new(io::stdin(), gff::GffType::GFF3);
 //! ```
 
 use std::io;
 use std::fs;
 use std::path::Path;
 use std::convert::AsRef;
 use std::collections::HashMap;
+use itertools::Itertools;
 
 use csv;
 
 use io::Strand;
 
+/// GffType
+///
+/// We have three format in the GFF family.
+/// The change is in the last field of GFF.
+/// For each type we have key value separator and field separator
+#[derive(Clone, Copy)]
+pub enum GffType {
+    /// Attribute format is key1=value, key2=value
+    GFF3,
+    /// Attribute format is key1 value; key2 value
+    GFF2,
+    /// Same as GFF2 just possible keyword and possible value change
+    GTF2,
+    /// Any, first field of tuple is key value separator, second is field separator
+    Any(u8, u8),
```

Comment:
Sorry to bother you again. But can you also remove `Any`? It is already a pity that the attribute format in gff/gtf is so heterogeneous. I don't want to encourage people to even use other delimiters.